### ISSUE TITLE
Provide tools for creating DOM-tree 

### ISSUE THREAD

**Issue by davyie** [davyie/SDA-js_labs#10]
In exercise 2 students have to create a DOM-tree. Please provide some tools to use for creating such a tree. Otherwise, the students might get confused.

**Comment by kwabe007** [comment]
Good point. They can do it by pen and paper as well so we should probably write that in the lab description.
@nour95 What tool did you use to make the DOM-tree?
